### PR TITLE
Fix fzf grep

### PR DESCRIPTION
**Pull request by basbebe**

### CODE REVIEW

File: share/kcr/commands/fzf/kcr-fzf-grep [hunk 1]
```diff
@@ -11,10 +11,13 @@
 
 # Not pretty but robust.
 # https://github.com/junegunn/fzf/issues/2581
-export PATHS_PATH=$(mktemp)
+export PATHS_PATH=$(mktemp -d)
```
Comment (alexherbo2) [inline share/kcr/commands/fzf/kcr-fzf-grep:14]: Have you tested the code?  `-d` will create a directory and paths will be written to the directory.

Comment (basbebe) [reply share/kcr/commands/fzf/kcr-fzf-grep]: apparently I misunderstood you yesterday and this is nonsense
removed commit

Comment (basbebe) [reply share/kcr/commands/fzf/kcr-fzf-grep]: removed `export`

**Comment by alexherbo2** [comment]
Thanks!

**Pull request merged**